### PR TITLE
 Tabs changing along with changing of page 

### LINKED ISSUE #179: Tabs don't change while going back, though the page changes.
If we switch tabs(like projects, mentors, FAQ) on http://projects.coala.io/#/projects and then try to go back, the black highlighter stays on the latest page only and doesn't move while the page changes.

Like as shown in the image although we are on the projects page but mentors tab is highlighted.

ment

### PR DESCRIPTION
**Pull request by AkshJain99**
Highlighting line under tabs are changing
along with changing of page.

Fixes #179

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: partials/tabs/projects.html [hunk 1]
```diff
@@ -91,6 +91,7 @@
 
 <script>
   $(document).ready(function(){
+  $('ul.tabs').tabs();
```
Comment (li-boxuan) [inline partials/tabs/projects.html]: Missing blank spaces.

File: partials/tabs/faq.html [hunk 2]
```diff
@@ -13,7 +13,8 @@
 </div>
 <script>
 $(document).ready(function() {
-  $('.collapsible').collapsible();
+$('ul.tabs').tabs();
+$('.collapsible').collapsible();
```
[the remark this answers is not in the document]
Comment (li-boxuan) [reply partials/tabs/faq.html]: ↑ Not addressed.

[the remark this answers is not in the document]
Comment (AkshJain99) [reply partials/tabs/faq.html]: i am not getting which spaced i removed?

[the remark this answers is not in the document]
Comment (li-boxuan) [reply partials/tabs/faq.html]: Compare https://github.com/coala/projects/pull/697#discussion-diff-240550396L16 to https://github.com/coala/projects/pull/697#discussion-diff-240550396R17

**Comment by AkshJain99** [comment]
Is it correct now @li-boxuan

File: partials/tabs/faq.html [hunk 3]
```diff
@@ -13,7 +13,8 @@
 </div>
 <script>
 $(document).ready(function() {
-  $('.collapsible').collapsible();
+$('ul.tabs').tabs();
```
[the remark this answers is not in the document]
Comment (AkshJain99) [reply partials/tabs/faq.html]: okay got it @li-boxuan  done :+1:

File: partials/tabs/faq.html [hunk 4]
```diff
@@ -14,6 +14,8 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
+    $('ul.tabs').tabs();
+
```
Comment (utkarsh2102) [inline partials/tabs/faq.html]: Is there a reason for leaving this line?

File: partials/tabs/faq.html [hunk 5]
```diff
@@ -14,6 +14,6 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
+    $('ul.tabs').tabs();
 });
-
```
Comment (li-boxuan) [inline partials/tabs/faq.html]: This removal has nothing to do with this issue, so don't do it. We can make it a newcomer issue 👍

Comment (AkshJain99) [reply partials/tabs/faq.html]: but here i added tab script line 17 which is needed to fix the bug @li-boxuan

**Comment by AkshJain99** [comment]
@li-boxuan  look at my changes are these changes fine because bug is fixed with these changes .

File: partials/tabs/faq.html (hunk 4 above)
Comment (li-boxuan) [reply partials/tabs/faq.html]: If Travis is failing coz there is no newline here, why isn't it failing https://github.com/coala/projects/pull/697/files#diff-b5dc5a5e22c39da3a9de7f98539bde5eR78?

File: partials/tabs/faq.html [hunk 6]
```diff
@@ -14,6 +14,7 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
-});
+    $('ul.tabs').tabs();
```
Comment (li-boxuan) [inline partials/tabs/faq.html]: The indentation is not correct.

File: partials/tabs/projects.html [hunk 7]
```diff
@@ -91,10 +91,11 @@
 
 <script>
   $(document).ready(function(){
-    $('.explore').click(function () {
-      $('html,body').animate({
-        scrollTop: $(".showcase").offset().top},
-        'slow');
+    $('ul.tabs').tabs();
+      $('.explore').click(function () {
```
Comment (li-boxuan) [inline partials/tabs/projects.html]: Why extra 2 spaces indentation here?

Comment (AkshJain99) [reply partials/tabs/projects.html]: that means travis is failing due to indentation  problem @li-boxuan ?

File: partials/tabs/faq.html (hunk 6 above)
Comment (AkshJain99) [reply partials/tabs/faq.html]: but then why is this showing @li-boxuan :sweat_smile: 
```
$ docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci
Unable to find image 'coala/base:latest' locally
latest: Pulling from coala/base
Status: Downloaded newer image for coala/base:latest
Executing section global...
[ERROR][06:03:24] The bear InvalidLinkBear does not fulfill all requirements. You are not connected to the internet.
Executing section commit...
Executing section linecount...
Executing section filenames...
Executing section markdown_filenames...
Executing section spacing...
|    | [NORMAL] SpaceConsistencyBear:
|    | Line contains following spacing inconsistencies:
|    | - No newline at EOF.
|----|    | /app/partials/tabs/faq.html
|    |++++| /app/partials/tabs/faq.html
|  17|  17|       $('ul.tabs').tabs();
|  18|  18| 
|  19|  19|   });
|  20|    |-</script>
|    |  20|+</script>
[INFO][06:03:26] Applied 'ShowPatchAction' on 'partials/tabs/faq.html' from 'SpaceConsistencyBear'.
Executing section html...
Executing section yml...
Executing section new_linelength_projects...
Executing section old_linelength_projects...
Executing section cli...
The command "docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci" exited with 1.
cache.2
store build cache
$ bundle clean
0.00s1.42snothing changed, not updating cache
Done. Your build exited with 1.
```

**Comment by jayvdb** [comment]
@KVGarg , is this still needed after the recent improvements?

**Comment by KVGarg** [comment]
No! As we're now a navbar in-place of the tab sliders

**Pull request closed without merging**